groupie/src/main/java/com/genius/groupie/Item.java

```diff
@@ -93,13 +78,11 @@ public void bind(T viewBinding, int position, List<Object> payloads) {
         return this == item ? 0 : -1;
     }
 
-    public boolean isClickable() {
```

Comment:
I think true is fine.  If you set the clicklistener then you would expect your items to be clickable by default, I think.  If you do not then it doesn't matter!